src/shared/icons/profile.svg

```diff
@@ -0,0 +1,4 @@
+<svg xmlns="http://www.w3.org/2000/svg" fill="none" viewBox="0 0 16 16">
```

Comment:
have the same icon in #377 named `new-channel`, we should decide on naming for that and then remove it in PR that will be merged later

Reply:
actually, it's not the same icon.  
![image](https://user-images.githubusercontent.com/51168865/112149421-3d755680-8bdf-11eb-899f-270d13625443.png))
![image](https://user-images.githubusercontent.com/51168865/112149506-5978f800-8bdf-11eb-81e1-3da0ad759d2a.png)